flake-module.nix

```diff
@@ -228,7 +228,7 @@ ihpFlake:
                     fi
 
                     ${pkgs.nixos-rebuild}/bin/nixos-rebuild switch -j auto --use-substitutes --fast --flake .#$1 --target-host $1 --build-host $1 --option substituters https://digitallyinduced.cachix.org --option trusted-public-keys digitallyinduced.cachix.org:digitallyinduced.cachix.org-1:y+wQvrnxQ+PdEsCt91rmvv39qRCYzEgGQaldK26hCKE=
-                    ssh $1 systemctl start migrate
+                    ssh $1 nix-shell --run migrate
```

Comment:
@mpscholten It seems `systemctl` is out of question here. I SSHed into the AMI image referred in the docs, and there's no such system service.